serviceAccountKey.json

```diff
@@ -0,0 +1,13 @@
+{
+  "type": "service_account",
+  "project_id": "truecrime-6f10f",
+  "private_key_id": "9ecb7d35e4fd982277fa77c9acac93da75e953d7",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDJWhriAEQPDfU/\n4rnLmJJ48ErDEe/CgDLJ+FL5sikNRG0i/l6MRV7LyCE0nDqcm5lWr2YX5HHH1kfR\nAcCjrubUnZgmRhAoVVTk3WYoulmcoPjoLfe+9CTzIZEwy+zn5lv9rgW00rdz/DB/\nRI7hYhTxnpLt7HresO8LFPxdF0tTBqDZtgbDX3edJ3LIMKPP/E+Wi0QJypZAX4jg\nobTpqPp2EScnWdC6PApA7JEb23DALtRelfqPXQRU2nZVKKUoYW9dPQMWr0i6QAiF\noeqc0zlhRMW2KBEfnOimKrvDgy1B1RNLus7O6L0DKpnFVe5AI0HUrr21RRxdvUXO\nNC/lbc9nAgMBAAECggEACaDR5URe/MX4QsL24HSeGG4Lx9KClQSSqSlt36Z0eRUb\nD4/QpJjSmiBVRNHtqZHKiM8gAid4hgufPoV8w+GirxW5cfxqsbW24VocPvlTkf5O\njgeuBPG2wfDfzlC0tjgZO62ydH2ZASZU3q5qvf312MHyQ7H0WX67cjEMNPAJYuG6\nCWV3eY4BMocD62UUwAJpqw8FiNxZyG+NcFP2LaC3l24V/7BXBeuVAPoi/b6cpzW6\nM+Tlsxh2tUJq18vpIEze5feJFcFzkkwwgMlMJNsxuRPJKO5zxDFkSQMi+rrXWN0J\n5HOlUEbjc55LDPAug0EtWAfRP+nXCizQPJyPc7Qx8QKBgQDpvcE25zdhHLZozW+8\n1zLdPCud3TGnGvN8Qp0ivvYdsEmCKwLYENtoa5ykE/J5z82cBP3HqQvBd2oWayka\n0dKuR9fTKBDqgh4zTwji2MenVtzHKBLDYgfWPdH1Zs0NAS4p4nZzaBQ8RCyh7sP0\nS1dpUi+j4yQ1Y7lz2u5MsfZX0QKBgQDchsA6cupdZEFLyZOe+5QI5Q12V1t+Are3\nnawIePci9W5Swb2xES6eDp6G6mXxk191E23V2zw3Dmo521B1scOF1NbMhRm/DhF0\n9VE3u2LA5Pb1R3erHOoHxMVQJ8Oqnf2581v7/sFa+livBUqAoDuIrrtIZ9R/4j3l\nB+OK0a+5twKBgQDZjXn2ZgJPavAEX6GufAWlkMlv5169kAIsF7w1iNN+EoR4mRvO\nivIMIIzs0nwPEHDo1GR4n0iHy2N8dkUN51lWSRwCNvp3qp/9VYSJzwcVI23eF9hO\n7kEpEOUIqaQUhJK4/UUOaULcTP/VwGu2RyGWr54Txef/oVANGypNTB5i4QKBgDOw\nPeQ1RSuIsSlTYQ91hjgjf+lKBn7dojQed9EwkTSXdDlfgde74EJZyNx0mlpqIYiu\neiMTFOUOsuVGu3Aqr1pyDbq4Q9WdNYhFa7j1aesuf5f0HV91EoGppFnqSBl4mJz6\nJohFKQrq/NhhFBpb9DhQRENvWsZyv/ErzTCff2G3AoGBAN8lgIZ0qPRnMWph1ouc\nTOcxdJLDnVhGLP/cn+/swHabwo8hdk3IoDj11DeukTmQBt8zDVL8twwcxR/7cAUn\nqo7FoULX32ZO9BC6tSEJKjH6IZZqbKDoRRkKEFU0rFN/Eac0qlrPvRdUf63aenMo\nQyKQ2iXpRsSQd9ChAjxUIlsz\n-----END PRIVATE KEY-----\n",
```

Comment:
This is a demo project but doing this is very very very wrong
Keys must not be in source

Reply:
I decided to remove it  and then I generated a new key, finally added that file to .gitignore